<?xml version='1.1' encoding='UTF-8'?>
<Tag plugin="workflow-support@3.8">
  <node class="cps.n.StepAtomNode" plugin="workflow-cps@2.92">
    <parentIds>
      <string>70</string>
    </parentIds>
    <id>71</id>
    <descriptorId>org.jenkinsci.plugins.workflow.steps.durable_task.ShellStep</descriptorId>
  </node>
  <actions>
    <cps.a.ArgumentsActionImpl plugin="workflow-cps@2.92">
      <arguments>
        <entry>
          <string>script</string>
          <string>
                    curl --location --request POST &apos;https://webexapis.com/v1/messages&apos; --header &apos;Authorization: Bearer ${webex}&apos; --header &apos;Content-Type: application/json&apos; --data-raw &apos;{
  &quot;roomId&quot;: &quot;Y2lzY29zcGFyazovL3VzL1JPT00vYWRhOWQ0ODAtZTYwNy0xMWViLTk3YWYtOTkwNTEzNzMzMjBl&quot;,
  
  &quot;markdown&quot;: &quot;Started at: 2021-07-22 14:01:53 UTC\nAuthor:Saurabh Kothari(saukotha@cisco.com)\nBuild ID: 11\nBuild_URL: http://192.168.192.250:8080/job/2.%20TenantOnboardingWithNAE/\nResult: FAILURE&quot; 
}&apos;
         </string>
        </entry>
      </arguments>
      <sensitiveVariables>
        <string>webex</string>
      </sensitiveVariables>
      <isUnmodifiedBySanitization>false</isUnmodifiedBySanitization>
    </cps.a.ArgumentsActionImpl>
    <wf.a.TimingAction plugin="workflow-api@2.46">
      <startTime>1626962734019</startTime>
    </wf.a.TimingAction>
    <s.a.LogStorageAction/>
    <wf.a.ErrorAction plugin="workflow-api@2.46">
      <error class="hudson.AbortException">
        <detailMessage>script returned exit code 127</detailMessage>
        <stackTrace>
          <trace>org.jenkinsci.plugins.workflow.steps.durable_task.DurableTaskStep$Execution.handleExit(DurableTaskStep.java:659)</trace>
          <trace>org.jenkinsci.plugins.workflow.steps.durable_task.DurableTaskStep$Execution.check(DurableTaskStep.java:605)</trace>
          <trace>org.jenkinsci.plugins.workflow.steps.durable_task.DurableTaskStep$Execution.run(DurableTaskStep.java:549)</trace>
          <trace>java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)</trace>
          <trace>java.util.concurrent.FutureTask.run(FutureTask.java:266)</trace>
          <trace>java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180)</trace>
          <trace>java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)</trace>
          <trace>java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)</trace>
          <trace>java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)</trace>
          <trace>java.lang.Thread.run(Thread.java:748)</trace>
        </stackTrace>
        <suppressedExceptions class="java.util.Collections$UnmodifiableRandomAccessList" resolves-to="java.util.Collections$UnmodifiableList">
          <c class="list"/>
          <list reference="../c"/>
        </suppressedExceptions>
      </error>
    </wf.a.ErrorAction>
  </actions>
</Tag>